smf/method/buckets-mdapi-setup

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+#
+# Copyright 2019 Joyent, Inc.
```

Comment:
2020

Reply:
Now I'm curious how the 'make check' stage of https://jenkins.joyent.us/job/joyent-org/job/manta-buckets-mdapi/job/PR-34/2/console passed.